packages/fast-components-react-msft/src/radio/examples.data.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from "react";
+import { IComponentFactoryExample } from "@microsoft/fast-development-site-react";
+import Radio from "./index";
+import { IRadioHandledProps } from "@microsoft/fast-components-react-base";
+import schema from "@microsoft/fast-components-react-base/dist/radio/radio.schema.json";
+import Documentation from "./.tmp/documentation";
+import { Label } from "../label";
+
+const slotRadioExample: JSX.Element = (
```

Comment:
these need to have an `htmlFor` attribute that points to the ID of the radio.

Reply:
added htmlFor